iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/TreeFactory.java

```diff
@@ -48,4 +50,8 @@ public MaintainerTree maintainer(SyntaxToken maintainerToken, List<SyntaxToken>
   public SyntaxToken argument(SyntaxToken token) {
     return token;
   }
+
+  public StopSignalTree stopsignal(SyntaxToken token, SyntaxToken tokenValue) {
```

Comment:
Why is the method name in lowercase only and the class name in PascelCase on the keyword?